amostra/tests/test_jsonschema.py

```diff
@@ -0,0 +1,40 @@
+import hypothesis_jsonschema
+from hypothesis import given, settings
+from hypothesis import strategies as st
+
+from amostra.utils import load_schema
+
+sample_dict = load_schema("sample.json")
+# Pop uuid and revision cause they are created automatically
+sample_dict['properties'].pop('uuid')
+sample_dict['properties'].pop('revision')
+sample_dict['required'] = ['name']
+sample_dict['properties']['name']['minLength'] = 3
+sample_dict['properties']['name']['maxLength'] = 5
+sample_dict['additionalProperties'] = False
+st_sample = hypothesis_jsonschema.from_schema(sample_dict)
+
+
+container_dict = load_schema("container.json")
+container_dict['properties'].pop('uuid')
+container_dict['properties'].pop('revision')
+container_dict['required'] = ['name', 'kind', 'contents']
```

Comment:
Same comments as above... it would be good if the _only_ change we had to make here was popping the read-only keys, `uuid` and `revision`.

Reply:
Same reason here. https://github.com/NSLS-II/amostra/pull/44/files#r319613117

Reply:
I think changing the schema before testing cuts against the spirit of how to use hypothesis. We _have_ to dispense with `uuid` and `required` become of how our API works, but it would be better not to mutate anything else about the schema.